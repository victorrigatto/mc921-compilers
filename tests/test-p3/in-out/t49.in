/* Bubble sort code */

int main() {
    int v[10];
    int n = 6, c, d, swap;
    for (c=0; c<n; c=c+1)
        v[c] = 10 - c;
    for (c=0; c<n-1; c=c+1)
        for (d=0; d<n-c-1; d=d+1)
            if (v[d] = v[d+1]) {
                swap = v[d];
                v[d] = v[d+1];
                v[d+1] = swap;
            }
    for (c=0; c<n; c=c+1)
        assert v[c] == c + 5;
    return 0;
}
