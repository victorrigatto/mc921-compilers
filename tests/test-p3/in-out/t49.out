SemanticError: The condition expression must be of type(bool) @ 10:17
